sdk/main/src/impl/HttpClient.cc

```diff
@@ -0,0 +1,91 @@
+#include "impl/HttpClient.h"
```

Comment:
Please include file header.

Reply:
Which header do you mean?  #include "impl/HttpClient.h" is in the HttpClient.cc.

Reply:
Sorry I meant license